todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailActivity.kt

```diff
@@ -33,10 +33,10 @@ class TaskDetailActivity : AppCompatActivity() {
         setContentView(R.layout.taskdetail_act)
 
         // Set up the toolbar.
-        setupActionBar(R.id.toolbar) {
```

Comment:
Keep as original; See https://kotlinlang.org/docs/reference/lambdas.html - "In Kotlin, there is a convention that if the last parameter to a function is a function, and you're passing a lambda expression as the corresponding argument, you can specify it outside of parentheses:"